README.md

```diff
@@ -6,6 +6,8 @@ This is a simple project to provide MAAS with KVM POD capability in a LXD contai
 - KVM must be installed and working. [KVM Install Guide](https://help.ubuntu.com/community/KVM/Installation)
 
 ## Running
+Ensure that the pool name in 'root-device' matches what you have in `lxd storage list`
```

Comment:
Is the command lxc storage list?
I think this only applies to versions newer than 2.0 (which is the default in xenial), probably best to add something to that affect here. "If using a version that supports storage pools (>2.0)..."